xbmc/input/KeyboardLayout.h

```diff
@@ -56,4 +58,5 @@ class CKeyboardLayout
   std::string m_language;
   std::string m_layout;
   Keyboards m_keyboards;
+  IInputCodingTable *m_codingtable;
```

Comment:
make this a std::unique_ptr since it's the only one owning it, the pointer in GUIDialogKeyboardGeneric can still be a plain pointer.